packages/examples/setupVitest.js

```diff
@@ -0,0 +1,3 @@
+import * as fs from "fs/promises";
+
+global.fetch = (url) => fs.readFile(url.href.replace("file://", ""));
```

Comment:
In order to get SolidJS to actually do stuff rather than compiling away all the interesting optimization code, we set the "browser" condition for our tests:

https://github.com/penrose/penrose/blob/4fe953fb26778e28debae7221de3f05ef30c4e2b/packages/examples/vite.config.ts#L4

[Rose interprets this](https://github.com/rose-lang/rose/blob/v0.4.5/packages/wasm/package.json#L9-L12) to mean "fetch the Wasm directly rather than embedding it as Base64," which Vite build dev mode and build mode are both fine with, but Vitest chokes on without this extra config.